### PR TITLE
15293 - Support for $0 service fees

### PR DESCRIPTION
**Pull request by seeker25**
*Issue #:*
https://github.com/bcgov/entity/issues/15293

*Description of changes:*
Add in TRF04 for sites registry. Rewrite ProductFee, ProductFeeViewEdit to use composition api.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/tests/unit/views/RequestNameView.spec.ts [hunk 1]
```diff
@@ -42,13 +42,13 @@ describe('RequestNameView.vue', () => {
...
   it('is a Vue instance', () => {
-    expect(wrapper.isVueInstance()).toBeTruthy()
+    expect(wrapper.vm).toBeTruthy()
   })
 
   it('renders the components properly', () => {
-    expect(wrapper.find(RequestNameView).exists()).toBe(true)
-    expect(wrapper.find(NumberedCompanyTooltip).exists()).toBe(true)
-    expect(wrapper.find(LearnMoreButton).exists()).toBe(true)
+    expect(wrapper.findComponent(RequestNameView).exists()).toBe(true)
```
Comment (seeker25) [inline auth-web/tests/unit/views/RequestNameView.spec.ts:49]: Depreciation message for vue-testing-tools

File: auth-web/tests/unit/views/RestrictedProductView.spec.ts [hunk 2]
```diff
@@ -38,7 +38,7 @@ describe('RestrictedProductView.vue', () => {
   })
 
   it('RestrictedProductView is a Vue instance', () => {
-    expect(wrapper.isVueInstance()).toBeTruthy()
+    expect(wrapper.vm).toBeTruthy()
   })
```
Comment (seeker25) [inline auth-web/tests/unit/views/RestrictedProductView.spec.ts:42]: Depreciation message for vue-testing-tools

**Comment by seeker25** [comment]
Found an issue with this don't merge yet

**Comment by seeker25** [comment]
Note: It's not possible to set service fees for GOVN accounts for BCOL. We're meeting tomorrow to discuss how to fix this.

**Review by kialj876 — APPROVED** [review]
nice work 👍 

thanks for changing all those findComponents / removing all the isVueInstance

**Pull request merged**